diskqueue.go

```diff
@@ -51,21 +57,22 @@ type Interface interface {
 	Delete() error
 	Depth() int64
 	Empty() error
+	TotalBytesFolderSize() int64
 }
 
 // diskQueue implements a filesystem backed FIFO queue
 type diskQueue struct {
 	// 64bit atomic vars need to be first for proper alignment on 32bit platforms
 
 	// run-time state (also persisted to disk)
-	readPos       int64
-	writePos      int64
-	readFileNum   int64
-	writeFileNum  int64
-	readMessages  int64
-	writeMessages int64
-	writeBytes    int64
-	depth         int64
+	readPos            int64
+	writePos           int64
+	readFileNum        int64
+	writeFileNum       int64
+	readMessages       int64
+	writeMessages      int64
```

Comment:
add comments: 
```
readMessages       int64 // Number of read messages 
writeMessages      int64 // Number of write messages
```
It might have two meanings: number of messages, bytes of messages

Reply:
Good catch!